gcloud/pubsub/connection.py

```diff
@@ -367,3 +367,23 @@ def subscription_delete(self, subscription_path):
         """
         path = '/%s' % (subscription_path,)
         return self.api_request(method='DELETE', path=path)
+
+    def subscription_modify_push_config(self, subscription_path,
+                                        push_endpoint):
+        """API call:  update push config of a subscription via a POST request
+
+        See:
+        https://cloud.google.com/pubsub/reference/rest/v1/projects.subscriptions/modifyPushConfig
+
+        :type subscription_path: string
+        :param subscription_path: the fully-qualfied path of the new
+                                  subscription, in format
+                                  ``projects/<PROJECT>/subscriptions/<TOPIC_NAME>``.
```

Comment:
**UPDATE**: Strange the view was different in the commit than on the PR. Or I'm just crazy.